src/logic/__tests__/operate.test.js

```diff
@@ -0,0 +1,28 @@
+import operate from '../operate';
+
+describe('Testing operate API sum', () => {
+  test('The next sum value must be 8', () => {
+    expect(parseInt(operate(3, 5, '+'), 5)).toBe(8);
+  });
+});
+
+describe('Testing operate API subtract', () => {
+  test('The next substract value must be 2', () => {
+    expect(parseInt(operate(5, 3, '-'), 5)).toBe(2);
+  });
+});
+describe('Testing operate API multiply', () => {
+  test('The next  multiply value must be 15', () => {
+    expect(parseInt(operate(3, 5, 'x'), 5)).toBe(15);
+  });
+});
+describe('Testing operate API division', () => {
+  test('The next divition value must be 2', () => {
+    expect(parseInt(operate(10, 5, '÷'), 5)).toBe(2);
+  });
+});
+describe('Testing operate API residue', () => {
+  test('The next residue value must be 2', () => {
+    expect(parseInt(operate(5, 3, '%'), 5)).toBe(2);
```

Comment:
- Please make sure all test pass.
- 
![Capture1](https://user-images.githubusercontent.com/63972670/200908219-14703676-925c-437b-ad2c-7eb8f3ae362e.PNG)

Reply:
Many thanks for your review @OLIPLICHE